tests/test-notebook/src/model.spec.ts

```diff
@@ -141,7 +141,7 @@ describe('@jupyterlab/notebook', () => {
           expect(called).to.be(true);
         });
 
-        it('should emit a `contentChanged` signal upon cell move', () => {
+        it.skip('should emit a `contentChanged` signal upon cell move', () => {
```

Comment:
Intentional?

Reply:
Nope, not intentional. Good catch. Thank you for taking a look over this.